ui/src/views/infra/HostAdd.vue

```diff
@@ -17,139 +17,197 @@
+                :value="cluster.id"
+                :key="cluster.id">
+                {{ cluster.name }}
+              </a-select-option>
+            </a-select>
+          </a-form-item>
+          <a-form-item>
+            <tooltip-label
+              slot="label"
+              :title="selectedClusterHyperVisorType === 'VMware' ? $t('label.esx.host') : $t('label.hostnamelabel')"
+              :tooltip="placeholder.url"/>
+            <a-input
+              v-decorator="['hostname', {
+                initialValue: hostname,
+                rules: [{ required: true, message: $t('message.error.required.input') }]
+              }]"
+              :placeholder="placeholder.url"></a-input>
+          </a-form-item>
+          <a-form-item v-if="selectedClusterHyperVisorType !== 'VMware'">
+            <tooltip-label slot="label" :title="$t('label.username')" :tooltip="placeholder.username"/>
+            <a-input
+              v-decorator="['username', {
+                initialValue: username,
+                rules: [{ required: true, message: $t('message.error.required.input') }]
+              }]"
+              :placeholder="placeholder.username"></a-input>
+          </a-form-item>
+          <a-form-item v-if="selectedClusterHyperVisorType !== 'VMware'">
+            <tooltip-label slot="label" :title="$t('label.password')" :tooltip="placeholder.password"/>
+            <a-input-password
+              v-decorator="['password', {
+                initialValue: password,
+                rules: [{ required: true, message: $t('message.error.required.input') }]
+              }]"
+              :placeholder="placeholder.password"></a-input-password>
+          </a-form-item>
+          <a-form-item v-if="selectedClusterHyperVisorType === 'Ovm3'">
+            <tooltip-label slot="label" :title="$t('label.agent.username')" :tooltip="$t('label.agent.username')"/>
+            <a-input
+              v-decorator="['agentusername', { initialValue: agentusername }]"
+              :placeholder="$t('label.agent.username')"></a-input>
+          </a-form-item>
+          <a-form-item v-if="selectedClusterHyperVisorType === 'Ovm3'">
+            <tooltip-label slot="label" :title="$t('label.agent.password')" :tooltip="$t('label.agent.password')"/>
+            <a-input
+              v-decorator="['agentpassword', { initialValue: agentpassword }]"
+              :placeholder="$t('label.agent.password')"></a-input>
+          </a-form-item>
+          <a-form-item v-if="selectedClusterHyperVisorType === 'Ovm3'">
+            <tooltip-label slot="label" :title="$t('label.agentport')" :tooltip="$t('label.agentport')"/>
+            <a-input
+              v-decorator="['agentport', { initialValue: agentport }]"
+              :placeholder="$t('label.agentport')"></a-input>
+          </a-form-item>
+          <a-form-item>
+            <tooltip-label slot="label" :title="$t('label.hosttags')" :tooltip="placeholder.hosttags"/>
```

Comment:
@utchoang can you please move the host tags and dedicated checkbox to the end of the form.

Reply:
This is to keep all host related details at one place, otherwise it is looking like this 

![image](https://user-images.githubusercontent.com/3348673/147721779-f0273510-3764-48bc-8816-f4a6d0cd28e8.png)